### ISSUE TITLE
New Order

### ISSUE THREAD

**Issue by s-monisha** [GreatDevelopers/LibreHatti#467]
Home » Catalog » Purchase orders » Add purchase order 
Can Tds be written as TDS or ,
TDS
(Tax deducted at source)
![new order](https://cloud.githubusercontent.com/assets/13537847/9832956/0252a444-59a5-11e5-96f1-2da0be30d74d.png)

**Comment by jasvir99** [comment]
Yeah sure. But don't change name of data element. Use verbose name only.
